### PR TITLE
More integer benchmarks for the jit: NDhrystone

### PR DESCRIPTION
**Pull request by AndyAyersMS**

### CODE REVIEW

**Comment by AndyAyersMS** [comment]
@cmckinsey PTAL

File: tests/src/JIT/Performance/CodeQuality/BenchI/NDhrystone/NDhrystone.cs [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+        m_ptrGlb.StringComp = "DHRYSTONE PROGRAM, SOME STRING".ToCharArray();
+        m_string1Loc = "DHRYSTONE PROGRAM, 1'ST STRING".ToCharArray();
+        m_array2Glob[8][7] = 10;  /* Was missing in published program */
+
+        for (i = 0; i < Iterations; ++i) {
+            Proc5();
+            Proc4();
+            intLoc1 = 2;
+            intLoc2 = 3;
+            m_string2Loc = "DHRYSTONE PROGRAM, 2'ND STRING".ToCharArray();
```
Comment (cmckinsey) [inline tests/src/JIT/Performance/CodeQuality/BenchI/NDhrystone/NDhrystone.cs]: Curious why you rewrote this to note use inline assignments? Not against it, but might have been more useful to test the optimizer the way it was before (successive array stores).

**Comment by cmckinsey** [comment]
Looks good otherwise.

File: tests/src/JIT/Performance/CodeQuality/BenchI/NDhrystone/NDhrystone.cs (hunk 1 above)
Comment (AndyAyersMS) [inline tests/src/JIT/Performance/CodeQuality/BenchI/NDhrystone/NDhrystone.cs]: This is closer to what the original C version did.

**Pull request merged**